tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -11,32 +11,32 @@ weave_options:
 Principal component analysis (PCA) is a fundamental technique to analyse and visualise data.
 It is an unsupervised learning method mainly used for dimensionality reduction.
 
-For example, we have a data matrix $\mathbf{X} \in \mathbb{R}^{N \times D}$, and we would like to extract $k \ll D$ principal components which captures most of the information from the original matrix.
-The goal is to understand $\mathbf{X}$ through a lower dimensional subspace (e.g. two-dimensional subspace for visualisation convenience) spanned by the principal components.
+For example, we have a data matrix $\textbf{X} \in \mathbb{R}^{N \times D}$, and we would like to extract $k \ll D$ principal components which captures most of the information from the original matrix.
```

Comment:
AFAIK it's not correct to use `textbf` here. IIRC `\mathbf{X}` also works fine already, doesn't it?

Reply:
Yes, both `textbf` and `mathbf` give bold texts, `mathbf` sounds more evidenced but not sure if it's rendered. let me change one thing each time and see how it works.